dtfs-central-api/api-tests/v1/utilisation-reports/fee-record-corrections/post-fee-record-correction.api-test.ts

```diff
@@ -177,7 +192,33 @@ describe(`POST ${BASE_URL}`, () => {
     const requestBody = aValidRequestBody();
 
     // Act
-    const response = await testApi.post(requestBody).to(getUrl(otherReportId, feeRecordId));
+    const response = await testApi.post(requestBody).to(getUrl(reportId, feeRecordId));
+
+    // Assert
+    expect(response.status).toEqual(HttpStatusCode.NotFound);
+  });
+
+  it(`should respond with a ${HttpStatusCode.NotFound} when the bank the report belongs to cannot be found`, async () => {
```

Comment:
this is fine for now, but wondering if it may be more readable if we put 'when the bank the report belongs to cannot be found' in a describe and line 203 to 216 in a beforeEach